src/qt/dcclient.cpp

```diff
@@ -27,6 +27,28 @@
 
 //##############################################################################
 
+namespace
+{
+    class QtPictureSetter
+    {
+    public:
+        QtPictureSetter(wxWindow *window, QPicture *pic)
+        {
+            m_window = window;
+            m_window->QtSetPicture( pic );
+        }
+
+        ~QtPictureSetter()
+        {
+            m_window->QtSetPicture( NULL );
+        }
+
+    private:
+        wxWindow *m_window;
```

Comment:
I'd make this const:
```suggestion
        wxWindow* const m_window;
```
and initialize it in the ctor initializer list then.